lightning/send-to-hapi.js

```diff
@@ -0,0 +1,20 @@
+fn(state => {
+  const { patient, encounter } = state.data;
+
+  return { ...state, patient, encounter };
+});
+
+fn(state => {
+  return post('Patient', state.patient, state => {
+    const { encounter } = state;
+
+    encounter.body.subject.reference = `Patient/${state.data.id}`;
+    return { ...state, encounter };
+  })(state);
+});
+
+fn(state => {
```

Comment:
this looks like an antipattern. go with `post('Encounter', state => state.encounter)` instead... no need for this `fn()` block